pettingzoo/mpe/simple/simple.py

```diff
@@ -51,8 +52,9 @@
 from .._mpe_utils.simple_env import SimpleEnv, make_env
 
 
-class raw_env(SimpleEnv):
+class raw_env(SimpleEnv, EzPickle):
     def __init__(self, max_cycles=25, continuous_actions=False, render_mode=None):
+        EzPickle.__init__(self, max_cycles, continuous_actions, render_mode)
         scenario = Scenario()
         world = scenario.make_world()
         super().__init__(
```

Comment:
When using multi-inheritance, don't use `super().__init__` as python needs to guess which of the classes you are you talking about. Therefore change to `SimpleEnv.__init__`

Reply:
Good catch I just left that code as is but definitely needed to adapt it